pymc3/parallel_sampling.py

```diff
@@ -346,8 +348,7 @@ def __init__(
         start_chain_num=0,
         progressbar=True,
     ):
-        if progressbar:
-            from tqdm import tqdm
+        from fastprogress import progress_bar
```

Comment:
i think this can go to the top of the file. if i recall, it was here and protected because of install difficulties with tqdm. 

If it stays here, I would leave it in a `if progressbar:` statement.